facts/pi/scripts/web.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
Would have expected this in `go` 😉 But cool this looks great!

Reply:
haha! if it were new maybe, but it's just a copy of the same script we use every year. They used to live in the scale-signs repo but seem more appropriate with the instructions.